tests/php/Unit/MollieWCHelperSettingsTest.php

```diff
@@ -0,0 +1,482 @@
+<?php # -*- coding: utf-8 -*-
+
+use function Brain\Monkey\Filters\expectApplied as expectFilterApplied;
+use function Brain\Monkey\Functions\expect;
+use Mollie\WooCommerce\Tests\TestCase;
+
+use Mollie_WC_Helper_Settings as Testee;
```

Comment:
Good job aliasing the SUT. Makes the code much more portable. Not to mention removing that long PSR-0 classname.